src/clj/puppetlabs/trapperkeeper/services/watcher/filesystem_watch_core.clj

```diff
@@ -82,51 +79,29 @@
                                           (filter dir-create?)
                                           (map #(.toPath (:path %)))))))
 
-(defn reset-key-and-unwatch-deleted!
-  [watch-key]
-  ;; reset key and remove from set if directory no longer accessible
-  (let [valid? (.reset watch-key)]
-    (when (not valid?)
-      ;; TODO: not sure if we need to do anything else here now that we're not retaining a map of
-      ;; these keys.  The Oracle example code makes it seem like maybe you do, but I can't tell what.
-      (log/info (trs "Removing watched directory {0} because it was deleted" (.watchable watch-key))))))
-
-;; This function (and other helper functions in this namespace)
-;; are heavily influenced by WatchDir.java from Java's official documentation:
-;; https://docs.oracle.com/javase/tutorial/essential/io/notification.html
```

Comment:
I felt like I was heavily changing the function that it was above and so it might no longer be applicable. In a new commit I've removed the bit about 'This function' and placed the note at the top of the functions section. The link to the oracle docs was very helpful!